core/App/ehsm_provider.cpp

```diff
@@ -810,4 +811,46 @@ ehsm_status_t ExportDataKey(ehsm_keyblob_t *cmk,
         return EH_OK;
 }
 
+std::string create_apikey()
+{
+    sgx_status_t sgxStatus = SGX_ERROR_UNEXPECTED;
+    sgx_status_t ret = SGX_ERROR_UNEXPECTED;
+    
+    uint32_t apikey_len;
+
+    // calculate length
+    ret = enclave_create_apikey(g_enclave_id,
+                                &sgxStatus,
```

Comment:
the length is 32， why need to calculat it?

Reply:
Consistent with other function

Reply:
no need to keep consistent with other function, due to the len is well known